### DISCUSSION TITLE
How do transition paths work?

### DISCUSSION THREAD

**Discussion by cjiminez** [senshu/Sozi#534] [Q&A]
Hi there, I'm really loving the power of Sozi, but I'm unable to find any documentation regarding transition paths. How do I create transition paths?

I've tried creating a couple simple transition arcs in illustrator with and without arrow heads, but Sozi freezes after I input the path names and try to preview the transition. I feel like there's some simple thing I'm missing. 

Would appreciate any help with this, thank you!

**Comment by aumouvantsillage** [comment]
There is no documentation for this feature yet. I hope to find time to work on the [Sozi guide](http://sozi.guide) during the summer.

In the meantime, you can find an example in the [Sozi-demos repository](https://github.com/sozi-projects/Sozi-demos).

There are two conditions to use a transition path:
* your drawing software must create an actual SVG `path` element (not a rectangle, ellipse or circle)
* the `path` element must have an ID attribute that you can use in Sozi

I am not familiar with Illustrator. Maybe we could move further if you could send me an example document.

**Reply by cjiminez** [reply]
Guillaume,

Thank you for getting back to me so quickly. I just tried it again and got it to work! I believe my problem is that I put arrow heads on my paths. This caused my path name to be applied to the group of line segment + arrow head. Then when I referenced the path name in Sozi, it would freeze up because it wasn't strictly a path element. Once I got rid of the arrow heads and made them into strict path elements, it worked. 

I noticed a weird interaction when the end of the path and the actual frame location don't line up. Seems like Sozi tries to find a way to follow the path while also ending on the final frame. I don't see this as a bug, rather more like Sozi trying to deal with conflicting information. 

I meant to upload the file on the original question, but ironically github doesn't accept SVG files. Here's what it looks like now:
![image](https://user-images.githubusercontent.com/86388654/123341073-2c035980-d502-11eb-9a7d-bc45ad7d4d5c.png)

Thanks again for your help and I look forward to using Sozi to make some exciting presentations!

**Reply by NedNet29** [reply]
Hi, you can rename `your_file.svg`to `yourfile.svg.txt`to upload it here
HTH
